frontend/model/contracts/group.js

```diff
@@ -41,34 +61,39 @@ DefineContract({
   name: 'gi.contracts/group',
   contract: {
     validate: objectOf({
-      // TODO: add 'groupPubkey'
-      groupName: string,
-      groupPicture: string,
-      sharedValues: string,
-      mincomeAmount: number,
-      mincomeCurrency: string,
-      proposals: objectOf({
-        [PROPOSAL_INVITE_MEMBER]: proposalSettingsType,
-        [PROPOSAL_REMOVE_MEMBER]: proposalSettingsType,
-        [PROPOSAL_GROUP_SETTING_CHANGE]: proposalSettingsType,
-        [PROPOSAL_PROPOSAL_SETTING_CHANGE]: proposalSettingsType,
-        [PROPOSAL_GENERIC]: proposalSettingsType
+      invites: mapOf(string, inviteType),
```

Comment:
Great! And on line 63 you can change `objectOf` => `objectMaybeOf` to help prepare for the future when this contract is frozen